src/pages/SearchPage.js

```diff
@@ -65,6 +65,7 @@ class SearchPage extends Component {
             props.personalDetails,
             '',
             props.betas,
+            true,
```

Comment:
we should not expose the config option outside. Instead of doing this create another wrapper method like `OptionsListUtils#getSearchOptions` which sets this flag to true and then use that method.

Reply:
After new updates all places using functions started sending true by default (getSearchOptions & getNewChatOptions). Now we dont need wrapper method. Because If I write a new function, the old one will not be used this time. I'm making the changes now as you said. I'm removing the parameter pass support. Thank you. 
After this i will update PR videos immediately.

Reply:
Sorry, But I didn't understand. Could you please rephrase this?

Reply:
`getSearchOptions` function only used on SearchPage. That's why I fixed the mistake I made by getting extra config. if i wrote another function the old function would be un used.